lib/soapy_cake/admin.rb

```diff
@@ -1,4 +1,5 @@
 module SoapyCake
+  # rubocop:disable Metrics/ClassLength
```

Comment:
I agree, it might even make more sense to split into logical separate parts (like `Affiliate`, `Advertiser`, `Reporting`) instead of the API schema CAKE provides, but yes, let's do that another time.